R/data_danica.R

```diff
@@ -0,0 +1,119 @@
+#' NBA Raptor
```

Comment:
Create a single help file that references both `raptor_by_player` and `raptor_by_team`. See @mariumtapal's example [here](https://github.com/rudeboybert/fivethirtyeight/blob/master/R/data_marium.R) on how to do so.